pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -615,6 +614,51 @@ were also defined on the `task` level.
 	          - echo $VAR_1
 	          - echo $APP_ENV
 
+### matrix
+
+The `martix` property allows you to define one or more environment variables
+with multiple values. After that, it creates multiple parallel jobs by
+calculating the Cartesian product of all sets of values.
```

Comment:
The `martix` property allows you to define one or more *environment variable sets*
(env var set contains multiple values for that env var).
In such setup `n` parallel jobs are created, where `n` equals cardinality of Cartesian product of all env var sets.

Reply:
Done.